SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -173,13 +173,21 @@ void USpatialConnectionManager::Connect(bool bInitAsClient, uint32 PlayInEditorI
 	bConnectAsClient = bInitAsClient;
 
 	const ISpatialGDKEditorModule* SpatialGDKEditorModule = FModuleManager::GetModulePtr<ISpatialGDKEditorModule>("SpatialGDKEditor");
-	if (SpatialGDKEditorModule != nullptr && SpatialGDKEditorModule->ShouldConnectToCloudDeployment() && bInitAsClient)
+	if (SpatialGDKEditorModule != nullptr && SpatialGDKEditorModule->ShouldConnectToCloudDeployment())
 	{
-		DevAuthConfig.Deployment = SpatialGDKEditorModule->GetSpatialOSCloudDeploymentName();
-		DevAuthConfig.WorkerType = SpatialConstants::DefaultClientWorkerType.ToString();
-		DevAuthConfig.UseExternalIp = true;
-		StartDevelopmentAuth(SpatialGDKEditorModule->GetDevAuthToken());
-		return;
+		if (bInitAsClient)
+		{
+			DevAuthConfig.Deployment = SpatialGDKEditorModule->GetSpatialOSCloudDeploymentName();
+			DevAuthConfig.WorkerType = SpatialConstants::DefaultClientWorkerType.ToString();
+			DevAuthConfig.UseExternalIp = true;
+			StartDevelopmentAuth(SpatialGDKEditorModule->GetDevAuthToken());
+			return;
+		}
+
+		if(SpatialGDKEditorModule->ShouldStartLocalServer())
```

Comment:
I know we currently return from the client block, but I think an else here might show the intention slightly more clearly.
```suggestion
		else if (SpatialGDKEditorModule->ShouldStartLocalServer())
```